chain/jsonrpc/client/src/lib.rs

```diff
@@ -199,6 +198,14 @@ impl JsonRpcClient {
     pub fn query(&mut self, request: RpcQueryRequest) -> RpcRequest<QueryResponse> {
         call_method(&self.client, &self.server_addr, "query", request)
     }
+
+    pub fn block_by_id(&mut self, block_id: BlockId) -> RpcRequest<BlockView> {
+        call_method(&self.client, &self.server_addr, "block", BlockQueryInfo::BlockId(block_id))
```

Comment:
It should be a list of one element (BlockId) passed as a parameter (at least, it used to be like that):

```diff
-        call_method(&self.client, &self.server_addr, "block", BlockQueryInfo::BlockId(block_id))
+        call_method(&self.client, &self.server_addr, "block", [BlockQueryInfo::BlockId(block_id)])
```